<?xml version="1.0"?>
<robot name="stewart">
 <link name="stewart_base_link">
   <visual>
     <geometry>
       <cylinder length="0.02" radius="0.18"/>
     </geometry>
   </visual>
 </link>

<link name="stewart_platform_link">
   <visual>
     <geometry>
       <cylinder length="0.02" radius="0.18"/>
     </geometry>
   </visual>
 </link>

 <joint name="movable_platform_joint" type="floating">
   <parent link="stewart_base_link" />
   <child link = "stewart_platform_link" />
   <origin xyz="0.0 0.0 0.2" rpy="0.0 0.0 0.0" />
 </joint>
</robot>
